climbing_stairs/climbing_stairs.py

```diff
@@ -2,13 +2,19 @@
 
 import sys
 
-def climbing_stairs(n, cache=None):
-  pass 
 
+def climbing_stairs(n, cache=None):
```

Comment:
Remember that the child in this problem also has the capability of climbing three stairs at a time.